hbp_knowledge/receptors/buckingham2009.bel

```diff
@@ -1466,74 +1585,80 @@ neurons (Fu and Jhamandas, 2003) and to evoke a alpha7-
 mediated calcium increases in presynaptic terminals
 isolated from rat hippocampus and neocortex (Dougherty
 et al., 2003)."
-
+SET Confidence = "High"
 a(CHEBI:"amyloid-beta polypeptide 42") -> act(p(HGNCGENEFAMILY:"Cholinergic receptors nicotinic subunits"))
 p(HGNC:APP, frag("25_35")) -> act(p(HGNCGENEFAMILY:"Cholinergic receptors nicotinic subunits"))
 p(HGNC:APP, frag("25_35")) -> act(p(HGNC:CHRNA7))
 act(p(HGNC:CHRNA7)) -> a(CHEBI:"calcium(2+)")
+UNSET Confidence
 
 SET Evidence="Perfusion of soluble Abeta into mouse
 prefrontal cortex increases dopamine secretion through
 a mechanism that is blocked by alpha7 antagonists (Wu et
 al., 2007)."
-
+SET Confidence = "High"
 SET Species="10090"
 SET MeSHAnatomy="Prefrontal Cortex"
 
 composite(a(CHEBI:"amyloid-beta"), p(HGNC:CHRNA7)) -> sec(a(CHEBI:dopamine))
 
 UNSET MeSHAnatomy
 UNSET Species
+UNSET Confidence
 
 SET Evidence="Again, despite numerous reports of a block of
 alpha7, one study indicated that Abeta failed to block alpha7, even
 though it blocked alpha4beta2, alpha2beta2 and alpha4alpha5beta2 receptors
 (Lamb et al., 2005)."
-
+SET Confidence = "High"
 a(CHEBI:"amyloid-beta") causesNoChange p(HGNC:CHRNA7)
 a(CHEBI:"amyloid-beta") -| act(complex(p(HGNC:CHRNA4),p(HGNC:CHRNB2)))
 a(CHEBI:"amyloid-beta") -| act(complex(p(HGNC:CHRNA2),p(HGNC:CHRNB2)))
 a(CHEBI:"amyloid-beta") -| act(complex(p(HGNC:CHRNA4),p(HGNC:CHRNA5),p(HGNC:CHRNB2)))
+UNSET Confidence
 
 SET Evidence="It has also been observed that although
 Abeta inhibits recombinant human and mouse alpha7
 nAChRs, transgenic mice overexpressing human Abeta express
 functional alpha7 nAChRs, and the amplitude of alpha7-
 mediated currents is no different from that of wild-type
 mice (Spencer et al., 2006)."
-
+SET Confidence = "High"
 SET Species="10090"
 
 a(CHEBI:"amyloid-beta") -| act(p(HGNC:CHRNA7, var("?")))
 a(CHEBI:"amyloid-beta") causesNoChange act(p(HGNC:CHRNA7))
 
 UNSET Species
+UNSET Confidence
 
 SET Evidence="The most comprehensive
 study of the effects of Abeta at different concentrations
 showed that at 10 pM, Abeta evoked an inward
 current mediated by rat alpha7 nAChRs expressed in X.
 laevis oocytes, whereas at 100 nM, Abeta blocked nicotine
 responses through desensitization (Dineley et al., 2002)."
-
+SET Confidence = "Low"
 SET Species="8355"
 
 a(CHEBI:"amyloid-beta") -| act(a(CHEBI:nicotine))
-
+#This rule can make things topsy turvy if concentrations are not a part of it.
```

Comment:
you are right, we should add concentrations as annotation here

Reply:
In form of comment is fine?